back-end/tests/service_test/loansService.test.js

```diff
@@ -24,7 +26,16 @@ test('Testing getLoansWithUserId can be call correctly', async () => {
         remaining: 3500,
         userId: 1
       },
-      type: undefined,
+      type: {
+        id: 1,
+        slug: 'student_1',
+        name: 'Student Loan',
+        duration: '5 years fixed',
+        icon: 'student',
+        title: '3% yearly interest rate',
+        description: 'Fixed low interests for students only',
+        interest: 0.03
+      },
```

Comment:
@JialinWang1 we should extract this data object to a const either in this file or in a separate mockData